lib/mindwendel_web/views/static_page.ex

```diff
@@ -1,3 +1,8 @@
 defmodule MindwendelWeb.StaticPageView do
   use MindwendelWeb, :view
+  alias Mindwendel.Brainstormings
+
+  def sorted_brainstormings_for(user) do
+    Brainstormings.list_brainstormings_for(user.id)
```

Comment:
This is just a minor nitpick - the method is basically just passing through the call, so it might be easier to user the same method name? sorted_brainstormings_for doesn't really sort anything, however list_brainstorming_for does.